src/command_modules/azure-cli-profile/azure/cli/command_modules/profile/custom.py

```diff
@@ -42,6 +42,24 @@ def show_subscription(subscription=None, expanded_view=None):
         return profile.get_expanded_subscription_info(subscription)
 
 
+def get_access_token(subscription=None, resource=None):
+    '''
+    get AAD token to access to a specified resource
+    :param resource: Azure resource endpoints. Default to Azure Resource Manager
+    Use 'az cloud show' command for other Azure resources
+    '''
+    profile = Profile()
+    creds, subscription, tenant = profile.get_raw_token(subscription_id=subscription,
+                                                        resource=resource)
+    return {
+        'tokenType': creds[0],
+        'accessToken': creds[1],
+        'expiresOn': creds[2]['expiresOn'],
+        'subscription': subscription,
```

Comment:
The `subscription` here has the return value from `get_raw_token`, which is an id. This command's client will need it to initialize the azure sdk client, so i always give out the `id`.
Let me adjust the naming across, it is a confusion which i didn't pay attention